deploy/awstools/afitools.py

```diff
@@ -92,23 +94,23 @@ def share_afi_with_users(afi_id, region, useridlist):
         )
     rootLogger.debug(result)
 
-def get_afi_sharing_ids_from_conf(conf):
+def get_afi_sharing_ids_from_conf(conf: Any) -> List[str]:
     usersdict = conf.ini['agfisharing']
     return usersdict.values()
 
-def share_agfi_in_all_regions(agfi_id, useridlist):
+def share_agfi_in_all_regions(agfi_id: str, useridlist: List[str]) -> None:
     """ For the given AGFI, for each fpga region, get the AFI, then share
     with the users in useridlist """
     all_fpga_regions = get_fpga_regions()
     for region in all_fpga_regions:
         afi_id = get_afi_for_agfi(agfi_id, region)
         share_afi_with_users(afi_id, region, useridlist)
 
-def firesim_tags_to_description(buildtriplet, deploytriplet, commit):
+def firesim_tags_to_description(buildtriplet: str, deploytriplet: str, commit: str) -> str:
     """ Serialize the tags we want to set for storage in the AGFI description """
     return """firesim-buildtriplet:{},firesim-deploytriplet:{},firesim-commit:{}""".format(buildtriplet,deploytriplet,commit)
 
-def firesim_description_to_tags(description):
+def firesim_description_to_tags(description: str) -> Dict[Any, Any]:
```

Comment:
You probably mean `Dict[Hashable, Any]`.  I wonder if mypy would catch it for you if you tried to key the returndict with something that isn't hashable or if that will fall through to being a runtime error?